### ISSUE TITLE
Your website only mentions sync via dropbox - not onedrive and gdrive.

### ISSUE THREAD

**Issue by pperrin** [keeweb/keeweb#1601]
Just a note to say I think you'd do well to highlight these other sync platforms.

I already use keepass on android and windows sync'd via gdrive and was looking for a linux client that I could use. At first I skipped this app as it made a point of dropbox but no others, so I had assumed it was dropbox only... I came back to see if I could add gdrive to it... then found from the code that it is already there!

TL;DR - add gdrive and onedrive mentions to the section of the webpage that already talks about dropbox.

Cheers!

**Comment by antelle** [comment]
Thanks! Fixed
